mllib/src/main/scala/org/apache/spark/ml/feature/RFormula.scala

```diff
@@ -63,31 +61,26 @@ class RFormula(override val uid: String) extends Estimator[RFormulaModel] with R
    */
   val formula: Param[String] = new Param(this, "formula", "R model formula")
 
-  private var parsedFormula: Option[ParsedRFormula] = None
```

Comment:
I removed it because there are multiple places to set params. We need to parse the input formula just-in-time.